components/director/internal/domain/apptemplate/converter_test.go

```diff
@@ -301,6 +301,88 @@ func TestConverter_InputFromGraphQL(t *testing.T) {
 	}
 }
 
+func TestConverter_UpdateInputFromGraphQL(t *testing.T) {
+	// GIVEN
+	appTemplateInputGQL := fixGQLAppTemplateUpdateInput(testName)
+	appTemplateInputModel := fixModelAppTemplateUpdateInput(testName, "{\"name\":\"foo\",\"description\":\"Lorem ipsum\"}")
+
+	testCases := []struct {
+		Name               string
+		AppConverterFn     func() *automock.AppConverter
+		WebhookConverterFn func() *automock.WebhookConverter
+		Input              graphql.ApplicationTemplateUpdateInput
+		Expected           model.ApplicationTemplateUpdateInput
+		ExpectedError      error
+	}{
+		{
+			Name: "All properties given",
+			AppConverterFn: func() *automock.AppConverter {
+				appConverter := automock.AppConverter{}
+				appConverter.On("CreateInputGQLToJSON", appTemplateInputGQL.ApplicationInput).Return(appTemplateInputModel.ApplicationInputJSON, nil).Once()
+				return &appConverter
+			},
+			WebhookConverterFn: func() *automock.WebhookConverter {
+				conv := &automock.WebhookConverter{}
+				conv.On("MultipleInputFromGraphQL", []*graphql.WebhookInput(nil)).Return([]*model.WebhookInput(nil), nil)
+				return conv
+			},
+			Input:         *appTemplateInputGQL,
+			Expected:      *appTemplateInputModel,
+			ExpectedError: nil,
+		},
+		{
+			Name: "Empty",
+			AppConverterFn: func() *automock.AppConverter {
+				appConverter := automock.AppConverter{}
+				return &appConverter
+			},
+			WebhookConverterFn: func() *automock.WebhookConverter {
+				conv := &automock.WebhookConverter{}
+				conv.On("MultipleInputFromGraphQL", []*graphql.WebhookInput(nil)).Return([]*model.WebhookInput(nil), nil)
+				return conv
+			},
+			Input:         graphql.ApplicationTemplateUpdateInput{},
+			Expected:      model.ApplicationTemplateUpdateInput{},
+			ExpectedError: nil,
+		},
+		{
+			Name: "Error when converting",
```

Comment:
add one more case for the second converter error as well

Reply:
Actually, all webhook converters will be removed, bc applicaitonUpdateInput no does not support updating. Will do that in next commit.